<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MalzemeGirisFormu</class>
 <widget class="QWidget" name="MalzemeGirisFormu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>466</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ürün Girişi</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>341</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="verticalSpacing">
     <number>6</number>
    </property>
    <item row="3" column="0" alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Giriş Tarihi :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButtonKaydet">
      <property name="focusPolicy">
       <enum>Qt::FocusPolicy::WheelFocus</enum>
      </property>
      <property name="text">
       <string>Kaydet</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Ürün Miktarı :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QPushButton" name="pushButtonStokDurumu">
      <property name="text">
       <string>Stok Durumunu Göster</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QLabel" name="label_1">
      <property name="text">
       <string>Ürün Adı :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" alignment="Qt::AlignmentFlag::AlignRight">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Ürün Fiyatı :</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QTableWidget" name="tableWidget_Girisler">
      <row>
       <property name="text">
        <string>New Row</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>New Column</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1" alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QLineEdit" name="lineEdit_1"/>
    </item>
    <item row="1" column="1" alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item row="2" column="1" alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QLineEdit" name="lineEdit_3"/>
    </item>
    <item row="3" column="1" alignment="Qt::AlignmentFlag::AlignLeft">
     <widget class="QLineEdit" name="lineEdit_4"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
